js/package.json

```diff
@@ -1,148 +1,152 @@
-    "react-native-inappbrowser-reborn": "^3.5.1",
-    "react-native-interactable": "^2.0.1",
-    "react-native-linear-gradient": "^2.5.6",
-    "react-native-pager-view": "^5.4.9",
-    "react-native-permissions": "^3.1.0",
-    "react-native-popover-view": "^4.1.0",
-    "react-native-progress": "^5.0.0",
-    "react-native-qrcode-scanner": "^1.6.0",
-    "react-native-qrcode-svg": "git+https://github.com/sakul-budhathoki/react-native-qrcode-svg.git#feat/circles",
-    "react-native-reanimated": "^2.2.4",
-    "react-native-restart": "^0.0.22",
-    "react-native-safe-area-context": "^3.3.2",
-    "react-native-screens": "^3.9.0",
-    "react-native-share": "^7.2.1",
-    "react-native-svg": "^12.1.0",
-    "react-native-tab-view": "^3.1.1",
-    "react-native-v8": "^0.64.0-patch.0",
-    "react-native-vector-icons": "^9.0.0",
-    "react-native-webview": "^11.14.3",
-    "react-redux": "^7.2.6",
-    "redux": "^4.1.2",
-    "redux-persist": "^6.0.0",
-    "sha3": "^2.1.4",
-    "string.fromcodepoint": "^1.0.0",
-    "tlds": "^1.226.0",
-    "v8-android-jit": "^9.88.0"
-  },
-  "devDependencies": {
-    "@babel/core": "^7.16.5",
-    "@babel/node": "^7.16.5",
-    "@babel/runtime": "^7.16.5",
-    "@react-native-community/eslint-config": "^2.0.0",
-    "@types/base64-js": "^1.3.0",
-    "@types/color": "^3.0.1",
-    "@types/jest": "^27.0.3",
-    "@types/linkify-it": "^3.0.2",
-    "@types/lodash": "^4.14.178",
-    "@types/react-native-share": "^3.3.1",
-    "@types/react-test-renderer": "^17.0.1",
-    "@typescript-eslint/eslint-plugin": "^5.8.1",
-    "@typescript-eslint/parser": "^5.8.1",
-    "babel-jest": "^27.3.1",
-    "babel-plugin-inline-react-svg": "^2.0.1",
-    "babel-plugin-module-resolver": "^4.1.0",
-    "babel-plugin-transform-remove-console": "^6.9.4",
-    "eslint": "^7.14.0",
-    "eslint-config-react-app": "^7.0.0",
-    "jest": "^27.4.5",
-    "metro-config": "^0.66.2",
-    "metro-react-native-babel-preset": "^0.66.2",
-    "react-dev-utils": "^11.0.4",
-    "react-native-svg-transformer": "^0.14.3",
-    "react-test-renderer": "^17.0.2",
-    "typescript": "^4.5.4"
-  },
-  "jest": {
-    "preset": "react-native",
-    "moduleFileExtensions": [
-      "ts",
-      "tsx",
-      "js",
-      "jsx",
-      "json",
-      "node"
-    ],
-    "testPathIgnorePatterns": [
-      "/node_modules/",
-      "/packages/web-dev-app/",
-      ".gomobile-cache/"
-    ],
-    "modulePathIgnorePatterns": [
-      "/node_modules/",
-      "/packages/web-dev-app/",
-      ".gomobile-cache/"
-    ],
-    "transformIgnorePatterns": [
-      "\\./node_modules/react-native-elements"
-    ],
-    "moduleNameMapper": {
-      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/assetsTransformer.js",
-      "\\.(css|less)$": "<rootDir>/assetsTransformer.js",
-      "@berty-tech/(.*)": "<rootDir>/packages/$1"
-    }
-  }
+	"name": "@berty-tech/js",
```

Comment:
You should disable your editor auto format feature on package.json and/or install an editorconfig plugin.